svg/solid.svg

```diff
@@ -0,0 +1,36 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect width="20" height="20" fill="white"/>
```

Comment:
Modify the following attribute values from line 2
`width="100%"`
`height="100%"`